### ISSUE TITLE
[Suggestion] Add an header separator option to 'csvtk pretty'

### ISSUE THREAD

**Issue by apcamargo** [shenwei356/csvtk#123]
[`tsv-utils`](https://github.com/eBay/tsv-utils) provides an option to print aligned tables with a separator between the header and the rows. Although you can get a somewhat similar result with `csvtk csv2md`, the headers do not align. It is a pretty minor feature, but it would be nice to have in `csvtk`.

```bash

$ cat sample.tsv

scaffold	gene	r2	d_prime	distance
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.4342857142857145	0.9999999999999998	25
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.2250000000000001	0.7500000000000001	31
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.20189075630252096	0.5166666666666667	70
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.06447368421052647	0.4375	85
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.12121212121212124	1.0	6
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	1.0000000000000002	1.0000000000000002	16
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.23684210526315785	1.0	45
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.14736842105263168	1.0	60
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.002840909090909071	0.06250000000000001	39
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.08622448979591868	0.3250000000000001	54


###

$ tsv-pretty -H -u --precision 3 sample.tsv

scaffold                 gene                          r2  d_prime  distance
--------                 ----                          --  -------  --------
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.434    1.000        25
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.225    0.750        31
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.202    0.517        70
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.064    0.438        85
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.121    1.000         6
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  1.000    1.000        16
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.237    1.000        45
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.147    1.000        60
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.003    0.063        39
bin_13_scaffold_1020129  bin_13_scaffold_1020129_1  0.086    0.325        54

###

$cat sample.tsv | csvtk round -t -n 3 -a | csvtk -t pretty

scaffold                  gene                        r2      d_prime   distance
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.434   1.000     25.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.225   0.750     31.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.202   0.517     70.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.064   0.438     85.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.121   1.000     6.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   1.000   1.000     16.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.237   1.000     45.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.147   1.000     60.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.003   0.063     39.000
bin_13_scaffold_1020129   bin_13_scaffold_1020129_1   0.086   0.325     54.000

###

$ cat sample.tsv | csvtk round -t -n 3 -a | csvtk csv2md

scaffold	gene	r2	d_prime	distance
:-------------------------------------------------------------------
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.434	1.000	25.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.225	0.750	31.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.202	0.517	70.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.064	0.438	85.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.121	1.000	6.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	1.000	1.000	16.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.237	1.000	45.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.147	1.000	60.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.003	0.063	39.000
bin_13_scaffold_1020129	bin_13_scaffold_1020129_1	0.086	0.325	54.000

```

**Comment by apcamargo** [comment]
Thanks, @shenwei356!

**Comment by shenwei356** [comment]
binaries are available at #126
